api/answers/config/routes.json

```diff
@@ -0,0 +1,52 @@
+{
+  "routes": [
+    {
+      "method": "GET",
+      "path": "/answers",
+      "handler": "answers.find",
+      "config": {
+        "policies": []
+      }
+    },
+    {
+      "method": "GET",
+      "path": "/answers/count",
+      "handler": "answers.count",
+      "config": {
+        "policies": []
+      }
+    },
+    {
+      "method": "GET",
+      "path": "/answers/:id",
+      "handler": "answers.findOne",
+      "config": {
+        "policies": []
+      }
+    },
```

Comment:
Are these boilerplate routes generated by strapi?  Do we want these exposed since we are using graphql?

Reply:
Yes, it's generated by Strapi. I can check and see if Strapi allows me to turn off the REST API